### DISCUSSION TITLE
Change more user settings and system settings in edit extension?

### DISCUSSION THREAD

**Discussion by khomutovspace** [datenstrom/community#489] [Ask a question] [answered]
I think that it would be a very effective and important to have an ability to work with users (user.ini) (I mean, change smth, approve, switch roles and so on) and with system strings (at least to have a big text field, where custom global settings strings could be changed). I think it would be helpful at least because it wouldn't be needed to transfer site from local to we as much as before.

**Comment by BsNoSi** [comment]
Give me an example why this should be necessary besides the available tools every ftp-client comes along with. Allowing opening "user.ini" with edit ui opens diverse huge security risks, too. Therefore it should be a very, very, VERY reasonable reason, to do so.

Editing users requires secured structures for those data which are not part of the wonderful clean and easily to configure YELLOW CMS design. Introducing this would lead to a much more complicated handling to deal with, compareable to "doku.wiki" and so on.

**Comment by khomutovspace** [comment]
I am totally agree with you about the security questions, which are really important. But I think, that Edit extensions is more about user, not a developer. I mean, that when you build site, it is ok for you to work only with files, also while the site is already on server. But when you are working with it for your client, who needs to work with in the future, I think it would be a very good idea. I also understand that Yellow is not much as client friendly as WP for example. But why not? At least in that case?

**Comment by markseuffert** [comment, marked as answer]
I would write a specific "feature request", for example one that explains your idea and then let people vote for it. When something is popular it will be picked up eventually. Another possibility is to experiment with a prototype, for example fork the edit extension and then add the features you want to see. About WP, a guiding design principle for me is minimalism. Less but better. [[1]](https://www.vitsoe.com/eu/about/good-design) [[2]](https://volument.com/blog/minimalism-the-most-undervalued-development-skill)

If you would like to suggest ideas or experiment with features, go ahead and tell us what you need.
